cmd/cli/cmd/client.go

```diff
@@ -91,12 +91,47 @@ func SetupClientWithAuth(ctx context.Context) (*openlaneclient.OpenlaneClient, e
 		return nil, err
 	}
 
+	return clientWithCSRFToken(ctx, client, opts...)
+}
+
+// cloneClientWithCookies creates a new OpenlaneClient instance
+// with with the same config and cookies from the original client.
+func cloneClientWithCookies(client *openlaneclient.OpenlaneClient, opts ...openlaneclient.ClientOption) (*openlaneclient.OpenlaneClient, error) {
```

Comment:
This doesn't seem like something that should be a non-exported function inside of the CLI only given the context, we should create a exported function inside of `pkg/openlaneclient/client.go` that the CLI consumes